README.rst

```diff
@@ -512,18 +514,18 @@ only those exact fixers are enabled whether or not level is set.
 With the ``--config-file`` option you can specify the path to the
 ``.php_cs`` file.
 
-By using ``--using-cache`` option you can set if caching
-mechanism should be used.
+By using ``--using-cache`` option with ``yes`` or ``no`` you can set if caching
+mechanism should be used (enabled by default).
```

Comment:
You suggest here that option is `yes` by default, which is not.
Fact that caching is enabled by default is inside caching section.